README.md

```diff
@@ -0,0 +1,62 @@
+# Fedora CI pipeline library
+
+This git repository contains a library of reusable [Jenkins Pipeline](https://jenkins.io/doc/book/pipeline/) steps and functions that can be used in your Jenkinsfile to make your Fedora CI pipelines simple.
+
+## How to use this library
+
+This library is intended to be used as a [global shared library](https://jenkins.io/doc/book/pipeline/shared-libraries/#global-shared-libraries) configured in your Jenkins instance.
+
+It is recommened to pin down a specific version of the library and update cautiously.
```

Comment:
It is recommended

Reply:
Thanks for the review @sturivny. I think I am missing some plugin in vscode - I'd expect that IDE would mark typos like these :)

Reply:
It's a tricky thing actually, probably you added that words like variables names =)